openapi.yaml

```diff
@@ -1,279 +1,795 @@
-          $ref: '#/definitions/PetResource'
-  PetPostBody:
-    type: object
-    properties:
-      data:
-        type: object
-        properties:
-          type:
-            $ref: '#/definitions/PetType'
-          attributes:
-            type: object
-            properties:
-              name:
-                $ref: '#/definitions/PetResource/properties/attributes/properties/name'
-              owner:
-                $ref: '#/definitions/PetResource/properties/attributes/properties/owner'
-              species:
-                $ref: '#/definitions/PetResource/properties/attributes/properties/species'
-            required:
-              - name
-              - owner
-              - species
-            additionalProperties: false
-        required:
-          - type
-          - attributes
-        additionalProperties: false
-    required:
-     - data
-    additionalProperties: false
-  PetType:
-    type: string
-    enum: [pet]
-  ErrorObject:
-    properties:
-      status:
-        type: string
-        description: HTTP status code
-        example: '123'
-      title:
-        type: string
-        description: A short, user readable summary of the error
-        example: Not Found
-      code:
-        type: string
-        description: An application-specific error code
-        example: '1234'
-      detail:
-        type: string
-        description: A long description of the error that may contain instance-specific details
-      links:
-        properties:
-          about:
+            example: College of Engineering
+        website:
+          type: string
+          description: the website of the location
+          format: uri
+          example: https://example.oregonstate.edu
+        sqft:
+          type: number
+          description: the size in square feet of the location
+          example: 12345
+          nullable: true
+        calendar:
+          type: string
+          description: a link, if available, to a calendar file for the location
+          format: uri
+          example: https://example.oregonstate.edu/cal.ics
+        campus:
+          type: string
+          example: corvallis
+          description: the OSU campus where the location is located
+          enum:
+            - corvallis
+            - extension
+            - cascades
+            - hmsc
+            - other
+        type:
+          type: string
+          enum: [building, dining, other, parking, cultural-center]
+          description: the type of the building
+        giRestroomCount:
+          description: Number of gender inclusive restrooms available in the location.
+          type: number
+          format: integer
+        giRestroomLimit:
+          description: >-
+            If true, access to this location's restrooms may be limited.
+            A residence hall might restrict bathroom use or building access to
+            residents only.
+          type: boolean
+        giRestroomLocations:
```

Comment:
Same, what do you think if we merge all giRestroom attributes into one `giRestroom` object like:

```json
"giRestroom": {
  "limit": ...,
  "count": ...,
  "locations": ...
}
```

Reply:
I like this as well, I think it's a good approach.